src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -21,21 +21,23 @@ import org.broadinstitute.dsde.workbench.model.WorkbenchIdentityJsonSupport._
 import org.broadinstitute.dsde.workbench.model.google.GoogleModelJsonSupport._
 import org.broadinstitute.dsde.workbench.model.google._
 import org.broadinstitute.dsde.workbench.model._
-import spray.json._
+import spray.json.{RootJsonFormat, RootJsonReader, _}
+import ca.mrvisser.sealerate
 
 // Create cluster API request
-case class ClusterRequest(labels: Option[LabelMap] = Option(Map.empty),
+final case class ClusterRequest(labels: LabelMap,
                           jupyterExtensionUri: Option[GcsPath] = None,
                           jupyterUserScriptUri: Option[GcsPath] = None,
                           machineConfig: Option[MachineConfig] = None,
+                          properties: Map[String, String],
```

Comment:
this should also be an `Option[Map[String,String]]` because existing UIs won't know to put in an empty map. in general we want to make any new fields optional if possible

Reply:
spoke with Qi in person, this is handled in the json decoder now which is a better place for this logic to live. same goes for the above case